README.md

```diff
@@ -12,7 +12,7 @@ screwed cluster state or something.
 ## Usage
 
 Provided configuration should be usable out of the box with boot2docker
-that runs on IP `192.168.59.103`. Change IP address in `docker-compose.yml`
+that runs on IP `192.168.99.100`. Change IP address in `docker-compose.yml`
```

Comment:
may because this is default docker-machine vm ip?
![image](https://cloud.githubusercontent.com/assets/795466/10003962/d89cc19c-60b7-11e5-815b-9cc37b9a8083.png)